environment/src/main/resources/application.yml

```diff
@@ -74,8 +74,8 @@ environment:
   freeipa:
     url: http://localhost:8090
     contextPath: /freeipa
-    supportedPlatforms: AWS,AZURE
     synchronizeOnStart: true
+    supportedPlatforms: AWS,AZURE,GCP
```

Comment:
I had a doubt here, will UI start showing GCP credential creation if we enable this here? If so do we need UI change for the entitlement check before making this change?

Reply:
https://github.com/hortonworks/hortonworks-cloud/pull/2347